### PR TITLE
Added REP integration

### PR DESCRIPTION
**Pull request by DWD3**

### CODE REVIEW

**Review by benston3 — CHANGES_REQUESTED** [review]
Instead of return an error page to the participant if the REP crediting fails. I think we should log the error with WebstudyCreditResponse (so we can look it up later) and return an error message to the requesting page (so the participant can continue), and use an alert to tell the participant that the REP crediting has failed, and that they should contact the researcher.

File: templates/experiment_wrapper.html [hunk 1]
```diff
@@ -99,20 +111,42 @@
...
+        var _mark_rep_as_done = function(){
+            $.ajax({
+                type: 'GET',
+                url: '/mark-rep-as-done?survey_code=' + _survey_code + '&uid=' + _uid,
+				//async: false,
+                success: function (data) {
+                        //alert("Your REP has been marked as done");
+                },
+                error: function (jqXHR, textStatus, errorThrown){
+                    alert("Your REP has not been updated,Here is the error message:\n"+jqXHR.responseText);
```
Comment (DWD3) [inline templates/experiment_wrapper.html:140]: @benston3 Ben actually the current build already does what you described. If the mark_rep_as_done() method in python returns a 500 code plus the error message. _mark_rep_as_done method in JS will catch it and give the user an alert. The user will still be able to continue with the experiment(just tested it).

**Comment by DWD3** [comment]
> Instead of return an error page to the participant if the REP crediting fails. I think we should log the error with WebstudyCreditResponse (so we can look it up later) and return an error message to the requesting page (so the participant can continue), and use an alert to tell the participant that the REP crediting has failed, and that they should contact the researcher.

File: templates/experiment_wrapper.html (hunk 1 above)
Comment (benston3) [reply templates/experiment_wrapper.html:140]: Okay, excellent

**Pull request merged**